src/ert/gui/model/snapshot.py

```diff
@@ -317,18 +323,14 @@ def parent(self, child: Optional[QModelIndex] = None) -> Optional[QObject]:
         return self.createIndex(parent_item.row(), 0, parent_item)
 
     @override
-    def data(
-        self, index: QModelIndex, role: Qt.ItemDataRole = Qt.ItemDataRole.DisplayRole
-    ) -> Any:
+    def data(self, index: QModelIndex, role: int = Qt.ItemDataRole.DisplayRole) -> Any:
```

Comment:
Can we use the a union of all the predefined roles from lines 27-36 here instead of int for role?

Reply:
the qt stubs define that parameter as int, so we have to as well